Document how to create a frame with velocities from existing frame without velocities (#9110)
astropy/astropy: docs/coordinates/velocities.rst

    >>> icrs.radial_velocity # doctest: +FLOAT_CMP
    <Quantity 23.42 km / s>

Adding Velocities to Existing Frame Objects
===========================================

Another use case similar to the above comes up when you have an existing frame
object (or |skycoord|) and want an object with the same position but with

> velocities added.  The most conceptually straightforward way to do this is to

Reviewer: ```suggestion
velocities added. The most conceptually direct way to do this is to
```

use the differential objects along with
`~astropy.coordinates.BaseCoordinateFrame.realize_frame`. The following snippet

> accomplishes a straightforward case where the desired velocities are known in

Reviewer: ```suggestion
accomplishes a well-defined case where the desired velocities are known in
```

the Cartesian representation::

    >>> icrs = ICRS(1*u.deg, 2*u.deg, distance=3*u.kpc)
    >>> icrs # doctest: +FLOAT_CMP
    <ICRS Coordinate: (ra, dec, distance) in (deg, deg, kpc)
        (1., 2., 3.)>
    >>> vel_to_add = CartesianDifferential(4*u.km/u.s, 5*u.km/u.s, 6*u.km/u.s)
    >>> newdata = icrs.data.to_cartesian().with_differentials(vel_to_add)
    >>> icrs.realize_frame(newdata) # doctest: +FLOAT_CMP
    <ICRS Coordinate: (ra, dec, distance) in (deg, deg, kpc)
        (1., 2., 3.)
     (pm_ra_cosdec, pm_dec, radial_velocity) in (mas / yr, mas / yr, km / s)
        (0.34662023, 0.41161335, 4.29356031)>

> A same mechanism can also be used to add velocities even if full 3D coordinates

Reviewer 2: ```suggestion
A similar mechanism can also be used to add velocities even if full 3D coordinates
```

> are not available - e.g. for a radial velocity observation of an object where

Reviewer: ```suggestion
are not available (e.g., for a radial velocity observation of an object where
```

> the distance is unknown. However, it requires a slightly different way of

Reviewer: ```suggestion
the distance is unknown). However, it requires a slightly different way of
```
